apps/designer/jest.config.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
 module.exports = {
```

Comment:
q: any strong reason why this jest config is not reusing the shared one?

Reply:
we probably just forgot to move it once we created shared one

Reply:
seems like we even improved it in designer but not for other packages, I remember trying to make inline snapshots without success in a package and gave up, lets merge the 2 together and move it

Reply:
let's try to unify it in some followup PR then

Reply:
https://github.com/webstudio-is/webstudio-designer/issues/393